modules/core/src/test/scala/cats/tests/CatsEquality.scala

```diff
@@ -1,26 +1,7 @@
 /*
- * Copied verbatim from cats-testkit-scalatest (https://github.com/typelevel/cats-testkit-scalatest)
```

Comment:
This was inlined because there wasn't an up-to-date release of cats-testkit-scalatest but we should be able to reintroduce that dependency now